frontend/app/controllers/spree/orders_controller.rb

```diff
@@ -101,10 +101,6 @@ def accurate_title
       end
     end
 
-    def current_order_has_items
```

Comment:
Shouldn't we deprecate this controller action before removing it?

Reply:
That action is not in master. It's the second commit in this PR, removing the action introduced in the first one. It's discussed in https://github.com/solidusio/solidus/pull/4012#discussion_r607654632. I haven't squashed because I want to be sure that @kennyadsl is ok with the result.

Reply:
Oh sure, sorry 🤦